data/gegno/ships.txt

```diff
@@ -0,0 +1,770 @@
+# Copyright (c) 2021 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+ship Shale
+	sprite ship/shale
+	thumbnail thumbnail/shale
+	swizzle 0
+	attributes
+		category Transport
+		cost 180000
+		mass 121
+		bunks 7
+		"cargo space" 36
+		drag 1.7
+		"engine capacity" 58
+		"fuel capacity" 400
+		"heat dissipation" 0.8
+		hull 825
+		"outfit space" 135
+		"required crew" 1
+		shields 0
+		"weapon capacity" 12
+		weapon
+			"blast radius" 18
+			"shield damage" 40
+			"hull damage" 132
+			"hit force" 210
+	outfits
+		"C3 Scrap Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering"
+		"RG15 Torch Thruster"
+	engine 0 40
+	leak leak 50 60
+	explode "small explosion" 5
+	explode "tiny explosion" 10
+	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's design of a shuttle-ship, small and compact with the single purpose as a transport vessel, whether it be cargo, trade goods, travellers, or recruits to move off homeworld to pursue the efforts of the Vi and the Scin."
```

Comment:
A dash makes more sense here than a colon. A list typically follows a colon, and a dash marks a next iteration of a phrase (if that makes sense), so having the "have all been carried..." part after a colon seems kind of run-on-y. Either way the suggestion works out.

Reply:
How about this?
```suggestion
	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's design of a shuttle-ship, small and compact with the single purpose as a transport vessel. What exactly it transports varies widely - cargo, trade goods, travellers, or recruits for the war efforts of the Vi and the Scin have all been carried by this versatile ship.""
```